design/FY2022/NFP-Python-HVAC-Diagram.md

```diff
@@ -0,0 +1,85 @@
+# Python HVAC Diagram #
+
+Jason DeGraw, Piljae Im, and Niraj Kunwar
+
+Oak Ridge National Laboratory
+
+Original Date: November 18, 2021
+
+## Justification for New Feature ##
+
+The HVAC-Diagram program is a Fortran utility that can be used to generate
+an SVG representation of the HVAC system in a model based on the BND file
+generated by EnergyPlus. The program has limited flexibility and the SVG
+output is not correctly rendered by all SVG rendering libraries. The SVG format
```

Comment:
This is #6384. Internet Explorer still works, but Chrome and Edge are putting the labels are in the upper left corner. Presumably that's because Edge now uses the same thing that Chrome does.